stories/apps/launch/data/KitchenSinkApp.js

```diff
@@ -0,0 +1,671 @@
+// import FlagParamsApp from "./FlagParamsApp";
+// import NumberParamsApp from "./NumberParamsApp";
+// import InputParamsApp from "./InputParamsApp";
+// import OutputParamsApp from "./OutputParamsApp";
+// import ReferenceGenomeApp from "./ReferenceGenomeApp";
+// import SelectParamsApp from "./SelectParamsApp";
+// import TextParamsApp from "./TextParamsApp";
+
+// export default {
+//     description: "everything but....",
+//     requirements: [
+//         {
+//             step_number: 0,
+//         },
+//     ],
+//     deleted: false,
+//     disabled: false,
+//     name: "kitchen sink",
+//     system_id: "de",
+//     label: "kitchen sink",
+//     id: "1778b1d6-5a83-11ea-9e38-008cfa5ae621",
+//     app_type: "DE",
+//     groups: [
+//         ...InputParamsApp.groups,
+//         {
+//             id: "177d4ae8-5a83-11ea-9e38-008cfa5ae621",
+//             name: "",
+//             label: "Text/Numerical Input",
+//             parameters: [
+//                 ...TextParamsApp.groups[0].parameters,
+//                 ...FlagParamsApp.groups[0].parameters,
+//                 ...NumberParamsApp.groups[0].parameters,
+//             ],
+//             step_number: 0,
+//         },
+//         ...SelectParamsApp.groups,
+//         ...OutputParamsApp.groups,
+//         ...ReferenceGenomeApp.groups,
+//     ],
+// };
```

Comment:
Looks like the above can be removed?

Reply:
Oops, yep, some cruft that shouldn't have been committed 😅